net/portmapper/portmapper.go

```diff
@@ -28,16 +28,15 @@ import (
 	"tailscale.com/util/clientmetric"
 )
 
-// Debug knobs for "tailscaled debug --portmap".
-var (
+type DebugKnobs struct {
```

Comment:
Brad would say that this needs docs since it's public

Reply:
We do really need a 🤖-Brad... added some 👍